src/NUnitFramework/framework/Internal/PropertyNames.cs

```diff
@@ -133,6 +133,10 @@ public class PropertyNames
         /// </summary>
         public const string IgnoreUntilDate = "IgnoreUntilDate";
 
+        /// <summary>
+        /// The optional Order the test will run in
+        /// </summary>
+        public const string Order = "Order";
         #endregion
```

Comment:
My mistake. My original thought was to add a property to work item and an entire new interface that allowed attributes to  modify work items, but that's obviously more complicated than we need.